portal-web/test/functional/com/liferay/portalweb/macros/analyticscloud/ACDXPSettings.macro

```diff
@@ -127,4 +127,18 @@ definition {
 		Click(locator1 = "ACDXPSettings#AC_GENERIC_ADD_OR_CONFIRMATION_BUTTON");
 	}
 
+	macro addUser {
```

Comment:
hey @ViniLopes20 , why are we encapsulating `JSONUser.addUser` and `User.logoutAndLoginPG`under `ACDXPSettings#addUser` and `ACDXPSettings#userLogin` respectively?

Just to keep them under our macros? 🤔

Reply:
Is a good point @alexcandidos, but I thought the test would be less polluted if they were in our macros. But thinking about it now I don't think that much changes, what do you think?